src/modules/app/containers/RouterFragment.js

```diff
@@ -2,26 +2,17 @@ import { Fragment } from 'redux-little-router'
 import React from 'react'
 import PropTypes from 'prop-types'
 import RouteConfig from '../RouteConfig'
-import { connect } from 'react-redux'
 import LivingFormPage from '../../../routes/living-form/containers/LivingFormPage'
-import LivingPage from '../../../routes/living/containers/LivingPage'
+import LivingPage from '../../../routes/categories/containers/CategoriesPage'
 import LivingManageOfferPage from '../../../routes/living-manage-offer/containers/LivingManageOfferPage'
```

Comment:
Wouldn't `LivingOfferPage` and `living-offer` as a name be enough? And why do we ust kebap-case here?

Reply:
Where do we use kebab-case? Folders are all in kebab-case